OpenRA.Game/Network/Order.cs

```diff
@@ -288,8 +283,7 @@ public byte[] Serialize()
 						w.Write(Target.FrozenActor.ID);
 						break;
 					case TargetType.Terrain:
-						// SerializableCell is guaranteed to be non-null if Type == TargetType.Terrain
-						w.Write(Target.SerializableCell.Value);
+						w.Write(Target.SerializablePos);
```

Comment:
A materially incorrect comment! `Target.FromPos` sets only the `WPos` field, not the `CPos cell` field. (`Target.FromCell` sets both).

Reply:
This will discard the cell Layer, which will eventually be important (move orders onto high bridges).  Can we calculate the cell if needed here instead?